src/styles/components/Button.scss

```diff
@@ -25,46 +25,153 @@
         border-color: $btn-default;
         size: $btn-size;
         height: $btn-height;
+        box-shadow: none !important;
+
+        &:focus{
+            outline: none !important;
+            border: $blue-450 $border-thick double;
+        }
+    
+        &:hover{     
+            border-color: $btn-hover;
+        }
+    
+        &:active{
+            border: $blue-600;
+            background-color: $blue-100;
+        }
+    
+        &:disabled{
+            border: $gray-500 $border-default solid;
+            background-color: $gray-200 !important;
+            color: $gray-450 !important;
+            font-size: 14px;
+        }
    
-    btn-link{
+    &.btn-link{
         @include link_button;
+        border: none !important;
+        outline-style: none !important;
+
+        &:focus{
+            border: $blue-450 $border-thin solid !important;
+        }
+    
+        &:hover{     
+            color: $blue-700;
+        }
+    
+        &:active{
+            color: $blue-800;
+            background-color: transparent !important;
+        }
+    
+        &:disabled{
+            border: none;
+            background-color: $gray-200 !important;
+            color: $gray-450 !important;
+            font-size: 14px;
```

Comment:
use rem instead of px

Reply:
disabled style is almost same and repeating for all buttons so check if you can include it like 
%btn-disabled {
           background-color: $gray-200 !important;
            color: $gray-450 !important;
            font-size: 14px;
}
......
 &:disabled {
     border: none;
     @extend %btn-disabled;
}